modules/support/client/components/BlockMemberLink.component.js

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
I think the `user` module would feel a more logical place for this component instead of the support module, what do you think?